pkg/security/module/module.go

```diff
@@ -112,7 +112,9 @@ func (m *Module) Init() error {
 	m.apiServer.Start(m.ctx)
 
 	// monitor policies
-	m.policyMonitor.Start(m.ctx)
+	if m.config.PolicyMonitorEnabled {
+		m.policyMonitor.Start(m.ctx)
+	}
```

Comment:
If this is a change in default user-visible behavior, we may want to add a changelog entry.

Reply:
This monitoring was added during this release and it not a user facing feature yet.